install-all-python.sh

```diff
@@ -0,0 +1,15 @@
+# because why not
+pip install wheel
+
+for file in $(find . -name "requirements.txt")
```

Comment:
The test requirements are actually annoyingly called requirements/tests.txt. I have a bug (#333) where I intended to move all the requirements files into requirements/ so they were easier to find but never got around to finishing it.

But this script would work to install all the prod reqs